lib/controller.js

```diff
@@ -185,7 +185,7 @@ class Controller extends DataService {
 
   setResData(data, scope, statusCode) {
     if (!statusCode) {
-      if (typeof data != 'undefined') {
+      if (data) {
```

Comment:
I think it's too general condition. If you wrote `return null;`, do you expect it to be "no content"? Or it means just returning `null` in body with status code `200`? The same about `return false;`

What do you think about this?